api/src/utils/get-column-path.ts

```diff
@@ -54,11 +55,13 @@ export function getColumnPath({ path, collection, aliasMap, relations }: ColPath
 		}
 
 		if (remainingParts.length === 1) {
-			return `${alias || parent}.${remainingParts[0]}`;
+			return { columnPath: `${alias || parent}.${remainingParts[0]}`, targetCollection: parent };
 		}
 
 		if (remainingParts.length) {
 			return followRelation(remainingParts, parent, alias);
 		}
+
+		return { columnPath: '', targetCollection: '' };
```

Comment:
Would returning `columnPath: ''` as default make the `if (!columnPath) continue;` in apply-query.ts line 358 not work as intended?

Reply:
It was previously undefined (void type), so I supposed the change into '' should be ok.